### DISCUSSION TITLE
Pocketbase - php sdk

### DISCUSSION THREAD

**Discussion by jonathan-martz** [pocketbase/pocketbase#1872] [Show and tell]
Iam currently work on a PHP Sdk to use Pocketbase Collection data or change them.
Iam more familiar with Laravel so Php is for me easier to use.

https://github.com/mkay-development/pocketbase-php-sdk

Should we make it a official one ?

**Comment by ganigeorgiev** [comment]
Thank you for sharing it.

There is nothing wrong if the community packages reside outside of the [`github.com/pocketbase`](https://github.com/pocketbase) org/namespace.

There are no plans to add official support for other SDKs until v1 is reached since it will be hard to maintain and support (_I'm already having difficulties keeping the JS and Dart changes in sync and it will be too much for me while still polishing the APIs_).

_For better discoverability I'll consider sometime in the future to create a section in the website or in the main repo readme to list all community maintained integrations, but for now you can file an issue/pr in https://github.com/benallfree/awesome-pocketbase if you want to list it there._
